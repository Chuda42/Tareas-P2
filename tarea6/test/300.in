# Implementar structs, las operaciones de crear y liberar
reiniciar
# Prueba de esVacioBinario, insertarEnBinario, raiz, izquierdo, derecho, buscarSubarbol
buscarSubarbol 1
esVacioBinario
insertarEnBinario (1,1.1)
esVacioBinario
raiz
buscarSubarbol 2
buscarSubarbol 1
insertarEnBinario (2,2.2)
insertarEnBinario (3,3.3)
izquierdo
derecho
buscarSubarbol 4
buscarSubarbol 1
buscarSubarbol 2
buscarSubarbol 3
reiniciar
# prueba de imprimirBinario
imprimirBinario
insertarEnBinario (5,5.5)
imprimirBinario
insertarEnBinario (3,3.3)
insertarEnBinario (6,6.6)
imprimirBinario
insertarEnBinario (2,2.2)
insertarEnBinario (7,7.7)
insertarEnBinario (9,9.9)
insertarEnBinario (8,8.8)
imprimirBinario
reiniciar
# Prueba de mayor, removerMayor, removerDeBinario
insertarEnBinario (10,10.10)
insertarEnBinario (9,9.9)
insertarEnBinario (11,11.11)
insertarEnBinario (8,8.8)
insertarEnBinario (12,12.12)
imprimirBinario
mayor
removerMayor
mayor
imprimirBinario
buscarSubarbol 10
buscarSubarbol 11
buscarSubarbol 9
removerDeBinario 9
imprimirBinario
removerMayor
imprimirBinario
removerDeBinario 10
removerDeBinario 8
imprimirBinario
reiniciar
# Prueba cantidadBinario, alturaBinario
insertarEnBinario (10,10.10)
insertarEnBinario (9,9.9)
insertarEnBinario (11,11.11)
insertarEnBinario (8,8.8)
insertarEnBinario (12,12.12)
imprimirBinario
cantidadBinario
insertarEnBinario (13,13.13)
imprimirBinario
cantidadBinario
alturaBinario
removerMayor
imprimirBinario
cantidadBinario
alturaBinario
reiniciar
# Prueba linealizacion
insertarEnBinario (16,16.16)
insertarEnBinario (8,8.8)
insertarEnBinario (24,24.24)
insertarEnBinario (4,4.4)
insertarEnBinario (12,12.12)
insertarEnBinario (20,20.20)
insertarEnBinario (28,28.28)
imprimirBinario
linealizacion
insertarEnBinario (22,-22.22)
imprimirBinario
reiniciar
# implementar las operaciones de iterador
estaDefinidaActual
agregarAIterador 5
agregarAIterador 2
agregarAIterador 1
agregarAIterador 8
reiniciarIterador
estaDefinidaActual
actualEnIterador
avanzarIterador
avanzarIterador
avanzarIterador
estaDefinidaActual
actualEnIterador
avanzarIterador
estaDefinidaActual
reiniciarIterador
estaDefinidaActual
actualEnIterador
Fin
