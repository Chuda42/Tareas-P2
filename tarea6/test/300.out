1>#  Implementar structs, las operaciones de crear y liberar.
2>Estructuras reiniciadas.
3>#  Prueba de esVacioBinario, insertarEnBinario, raiz, izquierdo, derecho, buscarSubarbol.
4>sub es vacío.
5>Vacio.
6>Insertado.
7>No vacío.
8>(1,1.10)
9>sub es vacío.
10>(1,1.10)
11>Insertado.
12>Insertado.
13>izquierdo de b es vacío.
14>(2,2.20)
15>sub es vacío.
16>(1,1.10)
17>(2,2.20)
18>(3,3.30)
19>Estructuras reiniciadas.
20>#  prueba de imprimirBinario.
21>
22>Insertado.
23>
(5,5.50)
24>Insertado.
25>Insertado.
26>
-(6,6.60)
(5,5.50)
-(3,3.30)
27>Insertado.
28>Insertado.
29>Insertado.
30>Insertado.
31>
---(9,9.90)
----(8,8.80)
--(7,7.70)
-(6,6.60)
(5,5.50)
-(3,3.30)
--(2,2.20)
32>Estructuras reiniciadas.
33>#  Prueba de mayor, removerMayor, removerDeBinario.
34>Insertado.
35>Insertado.
36>Insertado.
37>Insertado.
38>Insertado.
39>
--(12,12.12)
-(11,11.11)
(10,10.10)
-(9,9.90)
--(8,8.80)
40>(12,12.12)
41>Removido el mayor.
42>(11,11.11)
43>
-(11,11.11)
(10,10.10)
-(9,9.90)
--(8,8.80)
44>(10,10.10)
45>(11,11.11)
46>(9,9.90)
47>Removido.
48>
-(11,11.11)
(10,10.10)
-(8,8.80)
49>Removido el mayor.
50>
(10,10.10)
-(8,8.80)
51>Removido.
52>Removido.
53>
54>Estructuras reiniciadas.
55>#  Prueba cantidadBinario, alturaBinario.
56>Insertado.
57>Insertado.
58>Insertado.
59>Insertado.
60>Insertado.
61>
--(12,12.12)
-(11,11.11)
(10,10.10)
-(9,9.90)
--(8,8.80)
62>5.
63>Insertado.
64>
---(13,13.13)
--(12,12.12)
-(11,11.11)
(10,10.10)
-(9,9.90)
--(8,8.80)
65>6.
66>4.
67>Removido el mayor.
68>
--(12,12.12)
-(11,11.11)
(10,10.10)
-(9,9.90)
--(8,8.80)
69>5.
70>3.
71>Estructuras reiniciadas.
72>#  Prueba linealizacion.
73>Insertado.
74>Insertado.
75>Insertado.
76>Insertado.
77>Insertado.
78>Insertado.
79>Insertado.
80>
--(28,28.28)
-(24,24.24)
--(20,20.20)
(16,16.16)
--(12,12.12)
-(8,8.80)
--(4,4.40)
81>(4,4.40)(8,8.80)(12,12.12)(16,16.16)(20,20.20)(24,24.24)(28,28.28)
82>Insertado.
83>
--(28,28.28)
-(24,24.24)
---(22,-22.22)
--(20,20.20)
(16,16.16)
--(12,12.12)
-(8,8.80)
--(4,4.40)
84>Estructuras reiniciadas.
85>#  implementar las operaciones de iterador.
86>Actual de it NO está definida.
87>Agregando.
88>Agregando.
89>Agregando.
90>Agregando.
91>Reiniciado.
92>Actual de it está definida.
93>5
94>Avanzando.
95>Avanzando.
96>Avanzando.
97>Actual de it está definida.
98>8
99>Avanzando.
100>Actual de it NO está definida.
101>Reiniciado.
102>Actual de it está definida.
103>5
104>Fin.
